gatilegrid/tilegrids.py

```diff
@@ -112,6 +122,11 @@ class _MercatorBase:
     MINY = -20037508.342789244
     MAXY = 20037508.342789244
 
+    MINXCH = 572215.498516761
```

Comment:
we could reproject the lv95 or lv03 swissextent to epsg:3857 / epsg:4326, we would only have to define it once

Reply:
yes I tought about it, but I'd have to add pyproj here as well and re-project the same coordinates over and over again.
I think it is best to have them once and for all.